sdcm/cluster.py

```diff
@@ -3845,11 +3845,16 @@ def node_run_stress_bench(node, loader_idx, stress_cmd, node_list):
                     log_file=log_file_name)
             except Exception as exc:  # pylint: disable=broad-except
                 errors_str = format_stress_cmd_error(exc)
-                ScyllaBenchEvent(type='failure', node=str(node), stress_cmd=stress_cmd,
-                                 log_file_name=log_file_name, severity=Severity.CRITICAL,
-                                 errors=[errors_str])
+                if "truncate: seastar::rpc::timeout_error" in errors_str or "Operation timed out" in errors_str:
```

Comment:
I'm not exactly sure about this, if there is a timeout, during a run, we do want to stop a scylla bench if it's the main stress.

I don't know yet exactly how, I think we should have a way to set the priority/severity of specific stress command, and avoid this logic here.

Reply:
Yes, I did it after discussion with @roydahan . Timeout on truncate operation is the known issue and we don't want to stop and fail the test on it. 
"Operation timed out" - I am waiting the answer from Roy, but added meanwhile

Reply:
when do we get "Operation timed out"?

Reply:
> when do we get "Operation timed out"?

When failed to receive the answer from all nodes by CL

Reply:
but it's not the terminal failure that fails the stress, right?

Reply:
@roydahan 
It may fail load exactly like "truncate timeout". The question here if we want to fail the whole test

Reply:
So for now, as long as we aren't aware of a known issue with "Operation timeout" let's remove it.

Reply:
Done